project/Vagrantfile

```diff
@@ -1,17 +1,27 @@
 # set up the default terminal
 ENV["TERM"]="linux"
 
+# set minimum version for Vagrant
+Vagrant.require_version ">= 2.2.10"
 Vagrant.configure("2") do |config|
+  config.vm.provision "shell",
+    inline: "sudo su - && zypper update && zypper install -y apparmor-parser"
```

Comment:
Why is `apparmor-parser` needed?

Reply:
Makes sense, as it's required with the higher versions: https://github.com/rancher/k3os/issues/702